MetagenomeUtils.spec

```diff
@@ -43,13 +48,22 @@ module MetagenomeUtils {
 
     /*
       input_ref: BinnedContig object reference
+
+      optional params:
+      not_save_to_shock: not saving result bin files to shock
     */
     typedef structure {
       string input_ref;
+      boolean not_save_to_shock;
     } ExportParams;
 
+    /*
+      shock_id: saved packed file shock id
+      bin_file_list: a list of bin file path
+    */
     typedef structure {
       string shock_id;
+      list <string> bin_file_list;
```

Comment:
Does this point to the specific Fasta files?  Maybe it would be eaiser if we just point the directory containing all the files instead?  We may eventually put extra files in that directory too, and with this structure there isn't really a good way to distinguish files of different types (like if there was a summary file, or a file listing coverages, etc)